README.md

```diff
@@ -407,6 +407,25 @@ MAIL_TO = ["me@mydomain.com", "other_person@herdomain.com"]
 Krun uses `sendmail(8)` to send email, so you will need to make sure that this
 works prior to starting your experiment.
 
+You should disable any daemons which could interfere with your experiments
+(e.g. `cron`). You can turn off daemons before each process execution by adding
+the appropriate commands to `PRE_EXECUTION_CMDS` in your config file. E.g. for
+a systemd Linux system:
+
+```
+PRE_EXECUTION_CMDS = [
+    "sudo systemctl stop cron",
+    "sudo systemctl stop atd",
+    ...
+]
+```
+
+Similarly, you can use `POST_EXECUTION_CMDS` to turn daemons back on after each
+process execution.
+
+If you plan to turn on and off daemons using systemd, please make sure you have
+policykit installed.
```

Comment:
Shouldn't we just add this to the list of packages that people are asked to install?